base/applications/mspaint/winproc.cpp

```diff
@@ -911,44 +911,48 @@ LRESULT CMainWindow::OnCommand(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bH
             imageModel.NotifyImageChanged();
             break;
         case IDM_IMAGEROTATEMIRROR:
-            switch (mirrorRotateDialog.DoModal(mainWindow.m_hWnd))
             {
```

Comment:
Note that indentation here is broken.

Reply:
Fixed in https://github.com/reactos/reactos/commit/e4512e6af250d261c8ff4b0ece6b3411c503d6b0 .